crypto/src/hash/traits.rs

```diff
@@ -60,6 +60,12 @@ mod tests {
         const PREFIX: &'static HashPrefix = PROTOCOL_STATE;
     }
 
+    impl AsRef<[u8]> for TestHash {
```

Comment:
I guess TestHash was for testing and can be removed?

Reply:
it's used in the test cases here, so would be good to keep it